composer.lock

```diff
@@ -0,0 +1,10199 @@
+{
+    "_readme": [
```

Comment:
Please don't include composer.lock file, talked to @davidferlay about it and for now we wan't to let composer always install the latest version of packages according to composer.json constraints